src/main/java/com/sprint/mission/discodeit/config/MDCLoggingInterceptor.java

```diff
@@ -0,0 +1,48 @@
+package com.sprint.mission.discodeit.config;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.HandlerInterceptor;
+import org.springframework.web.servlet.ModelAndView;
+
+@Component
+@Slf4j
+public class MDCLoggingInterceptor implements HandlerInterceptor {
+
+  @Override
+  public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler)
+      throws Exception {
+
+    try {
+      UUID uuid = UUID.randomUUID();
+      MDC.put("requestId", uuid.toString());
+      MDC.put("requestMethod", request.getRequestURL().toString());
+      MDC.put("requestUrl", request.getMethod());
```

Comment:
requestMethod, requestUrl 값이 서로 뒤바뀐 것 같아요.
```suggestion
      MDC.put("requestMethod", request.getMethod());
      MDC.put("requestUrl", request.getRequestURL().toString());
```